Dockerfile

```diff
@@ -79,6 +74,13 @@ RUN ./caldp-install-fitscut   /usr/local && \
    echo "/usr/local/lib" >> /etc/ld.so.conf && \
    ldconfig
 
+USER developer
+RUN pip install --upgrade pip
+RUN pip install awscli boto3
+# RUN pip install jupyterlab
```

Comment:
can you remove the jupyterlab line completely?

Reply:
Yeah.   Possibly I can get rid of all of that just by making them caldp dependencies if they're not already.   I'll fool with it.

Reply:
in particular I'm a tiny bit worried now about overwriting something from the base env